viewer/src/datamodels/cad/CadNode.ts

```diff
@@ -45,14 +46,21 @@ export class CadNode extends THREE.Object3D {
   private readonly _events = {
     loadingHintsChanged: new EventTrigger<LoadingHintsChangeListener>()
   };
+  private _instancedMeshManager: InstancedMeshManager;
 
   constructor(model: CadModelMetadata, materialManager: MaterialManager) {
     super();
     this.type = 'CadNode';
     this.name = 'Sector model';
     this._materialManager = materialManager;
 
+    const instancedMeshGroup = new THREE.Group();
```

Comment:
```suggestion
    const instancedMeshGroup = new THREE.Group();
    instancedMeshGroup.name = 'InstancedMeshes';
```
For easier debugging